### DISCUSSION TITLE
Emebeddings for Excalidraw

### DISCUSSION THREAD

**Discussion by lumoe** [outline/outline#4940] [Ideas]
[Excalidraw](https://github.com/excalidraw/excalidraw)  is an awesome, yet simple piece of software that enables hand collaborative whiteboard drawing. An alternative to that is [tldraw]( https://github.com/tldraw/tldraw), where there already exists a FR: https://github.com/outline/outline/discussions/4174. There is also an implementation for the [Obsidian](https://github.com/zsviczian/obsidian-excalidraw-plugin) note-taking app for Excalidraw as a reference. 

As far as I am concerned, there is no possibility to have a public link share from the [website](https://excalidraw.com/)—so the implementation should work within outline.

**Comment by dustinblackman** [comment]
With Excalidraw having it's entire application within a [React component](https://www.npmjs.com/package/@excalidraw/excalidraw), I think this can be accomplished similar to the Mermaid plugin. With Excalidraw exporting its state as JSON, with all the necessary hooks, I believe it could support parallel editing just like Outline's documents do now.

**Comment by ovizii** [comment]
Obsidian has a couple of more features which could be implemented like its canvas for example.
